codex-rs/tui/src/lib.rs

```diff
@@ -139,6 +144,38 @@ pub async fn run_main(
         .with(tui_layer)
         .try_init();
 
+    #[allow(clippy::print_stderr)]
```

Comment:
Low pri, but I would consider moving this to `updates.rs` so you can have a smaller thing attached to `#[cfg(not(debug_assertions))]`.